pkg/config/setup/config.go

```diff
@@ -1483,6 +1483,7 @@ func serializer(config pkgconfigmodel.Setup) {
 	config.BindEnvAndSetDefault("serializer_use_events_marshaler_v2", true)
 
 	config.BindEnvAndSetDefault("use_v2_api.series", true)
+	config.BindEnvAndSetDefault("use_v3_api.series", false)
```

Comment:
Just curious, are these two flags mutually exclusive? Will a customer only ever be using v2 OR v3?

Reply:
In the end, when this is GA, most customers that only send to the intake would indeed pick one. In the context of this PR, I needed something to enable v3 to be able to test it. On a second thought, I should use something more placeholder-like here, to make sure we revisit this later.